src/main/resources/io/fabric8/jenkins/openshiftsync/GlobalPluginConfiguration/config.jelly

```diff
@@ -92,7 +91,5 @@
       description="Time in seconds the sync plugin runs a list operation for secrets">
       <f:textbox />
     </f:entry>
-    <f:validateButton title="Validate configuration" progress="Validating..." method="validate"
```

Comment:
if we remove this, we need to remove all the `validate` method.
But, the validation method is quite informative for sizing purposing and configuration of the cluster mode

Reply:
I'm fine leaving the validation in if we can figure out how to use it *AND* how to have the fields in question actually show up on the config panel.

They currently do not appear.

Any idea why that is the case @akram ?